From f925283662c7d00c1686121b13963150dc2acf3c Mon Sep 17 00:00:00 2001
From: Rostislav Lastochkin <remtrik@mail.ru>
Date: Sun, 17 Aug 2025 18:59:25 +0300
Subject: [PATCH] hwdb: Add Accelerometer mount matrix for Irbis TW43

(cherry picked from commit e196be154e9e7fc16b477d6cd09a58010990fb15)

Resolves: RHEL-72702
---
 hwdb.d/60-sensor.hwdb | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index d242025bd2..559fd4151e 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -662,6 +662,10 @@ sensor:modalias:acpi:BMA250*:dmi:*:bvritWORKS.G.WI71C.JGBMRB*:*:svnInsyde:pni71c
 # Irbis
 #########################################
 
+#TW43
+sensor:modalias:acpi:BMA250E*:dmi:*:svnIRBIS:pnTW43:*
+ ACCEL_MOUNT_MATRIX=0, -1, 0; -1, 0, 0; 0, 0, -1
+
 #TW90
 sensor:modalias:acpi:BOSC0200*:dmi:*:svnIRBIS:pnTW90:*
  ACCEL_MOUNT_MATRIX=0, 1, 0; -1, 0, 0; 0, 0, 1
